sphinxcontrib/jinja.py

```diff
@@ -58,10 +65,14 @@ def run(self):
                 print('\n'.join(self.content))
                 print('********** End Jinja Debug Output: Template Before Processing **********')
                 print('')
-            tpl = Environment(
+            env = Environment(
                       loader=FileSystemLoader(
                           self.app.config.jinja_base, followlinks=True)
-                  ).from_string('\n'.join(self.content))
+                  )
+            env.filters.update(self.app.config.jinja_custom_filters)
```

Comment:
we need some documentation for this feature.

Reply:
how is this working, what do I have to put in this jinja_custom_filters variable. please document here:

https://github.com/tardyp/sphinx-jinja#available-options